rabbitmq-operator/src/main/resources/log4j2-spring.xml

```diff
@@ -2,7 +2,7 @@
 <Configuration status="WARN">
     <Appenders>
         <Console name="Console" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %notEmpty{[cluster %X{cluster}]} %m%n"/>
+            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %notEmpty{[%notEmpty{resourceName: %X{resourceName}; }%notEmpty{clusterName: %X{clusterName}; }%notEmpty{namespace: %X{namespace}; }%notEmpty{type: %X{type}}] }%m%n"/>
```

Comment:
Do you think namespace should precede resourceName, as they're basically a hierarchy?

Reply:
Yeah, I think I had even planned on doing that but I forgot.